test/__mocks__/session.mock.ts

```diff
@@ -18,12 +18,15 @@ import {
   managingOfficerType,
   presenterType,
   trustType,
-  dueDiligenceType
+  dueDiligenceType,
```

Comment:
unnecessary change here too

Reply:
Resolved but note this is a branch that has 3 large branches in it, difficult to track changes

Reply:
No worries, I  agree ❤️